core/metacling/src/TClingClassInfo.cxx

```diff
@@ -116,15 +116,15 @@ TClingClassInfo::TClingClassInfo(cling::Interpreter *interp, const char *name, b
 TClingClassInfo::TClingClassInfo(cling::Interpreter *interp,
                                  const Type &tag)
    : TClingDeclInfo(nullptr), fInterp(interp), fFirstTime(true), fDescend(false), fIterAll(kTRUE),
-     fIsIter(false), fType(0), fTitle(""), fOffsetCache(0)
+     fIsIter(false), fType(nullptr), fTitle(""), fOffsetCache(0)
 {
    Init(tag);
 }
 
 TClingClassInfo::TClingClassInfo(cling::Interpreter *interp,
                                  const Decl *D)
    : TClingDeclInfo(nullptr), fInterp(interp), fFirstTime(true), fDescend(false), fIterAll(kTRUE),
-     fIsIter(false), fType(0), fTitle(""), fOffsetCache(0)
+     fIsIter(false), fType(nullptr), fTitle(""), fOffsetCache(0)
```

Comment:
```suggestion
     fIsIter(false), fOffsetCache(0)
```
Given the warning from clang-tidy: https://travis-ci.org/github/root-project/root/jobs/772600580